src/detours.h

```diff
@@ -381,7 +381,11 @@ extern const GUID DETOUR_EXE_RESTORE_GUID;
 extern const GUID DETOUR_EXE_HELPER_GUID;
 
 #define DETOUR_TRAMPOLINE_SIGNATURE             0x21727444  // Dtr!
-typedef struct _DETOUR_TRAMPOLINE DETOUR_TRAMPOLINE, *PDETOUR_TRAMPOLINE;
+typedef struct _DETOUR_TRAMPOLINE DETOUR_TRAMPOLINE, *PDETOUR_TRAMPOLINE;
+
+#ifndef DETOUR_MAX_SUPPORTED_IMAGE_SECTION_HEADERS
+#define DETOUR_MAX_SUPPORTED_IMAGE_SECTION_HEADERS      32
+#endif // !DETOUR_MAX_SUPPORTED_IMAGE_SECTION_HEADERS
```

Comment:
What number do you need here?
The original PE spec said 96, though that might be true any longer.
Can we safely grow it to 64 or 96??

Reply:
It is indeed 96 that is being used, based on [documentation ](https://docs.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-image_file_header) that claims that is the supported limit of the loader.

I would equally be fine with making the default 96, or making it 96 without making it configurable.

Reply:
96 I believe is derived from the kernel memory manager being willing to read up to two pages of headers.